desktop/src/lib/projects.ts

```
@@ -326,11 +362,61 @@ export const projectsApi = {
+        `/api/projects/${pid}/doc-review/${encodeURIComponent(docPath)}`,
+      ),
+    setState: (pid: string, docPath: string, state: string) =>
+      http<{ project_id: string; doc_path: string; review_state: string; reviewed_by: string | null; reviewed_at: number | null; changes_requested_by: string | null; changes_requested_at: number | null }>(
+        `/api/projects/${pid}/doc-review/${encodeURIComponent(docPath)}`,
+        { method: "PUT", body: JSON.stringify({ state }) },
+      ),
+    list: (pid: string, state?: string) =>
+      http<{ items: Array<{ project_id: string; doc_path: string; review_state: string; reviewed_by: string | null; reviewed_at: number | null; changes_requested_by: string | null; changes_requested_at: number | null }> }>(
+        `/api/projects/${pid}/doc-reviews${state ? `?state=${encodeURIComponent(state)}` : ""}`,
+      ).then((r) => r.items),
+  },
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)